engines/dfc_provider/lib/data_food_consortium/connector/importer.rb

```diff
@@ -120,18 +120,30 @@ def resolve_object(object)
       end
 
       def skos_concept(object)
-        return unless object.uri?
+        # Sometimes we get given a literal object with a value referring to skos concept
+        # ie "dfc-pt:drink"
+        return unless object.value.match(":") || object.uri?
 
         id = object.value.sub(
           "http://static.datafoodconsortium.org/data/measures.rdf#", "dfc-m:"
         ).sub(
           "https://github.com/datafoodconsortium/taxonomies/releases/latest/download/measures.rdf#",
           "dfc-m:"
+        ).sub(
+          "dfc-pt:",
+          "https://github.com/datafoodconsortium/taxonomies/releases/latest/download/productTypes.rdf#"
```

Comment:
@mkllnk I fixed the importer based on your comment here : https://github.com/datafoodconsortium/connector-codegen/pull/12#discussion_r1440030320